### DISCUSSION TITLE
PLFX Stack (nativewind monorepo starter)

### DISCUSSION THREAD

**Discussion by panzacoder** [nativewind/nativewind#783] [Show and tell]
👋 Just wanted share that I extracted out my most recent project setup to a starter project:

[PLFX Stack](https://github.com/panzacoder/plfx-stack)

Proud of where this is at, but still have some TODOs still need to:

- [ ] Move off of npm, my original project was using yarn v4, but when I extracted I had wild install issues. I tried yarn v3, yarn classic, pnpm, bun, and didn't have the time to debug any of them properly. I'll come back to this eventually.
- [ ] Get turbo tasks properly setup for builds and tests
- [ ] Fix `@expo/vector-icons` support in storybook
- [ ] Fix issues with `TextLink`. I tried combining internal and external links, but this doesn't behave like I would expect on native, so I probably need to take another approach.
- [ ] Figure out how to make the web nav not appear in the webview. Probably going to use parallel routes for this in NextJS

I would love your feedback, and hope you can use it in your next project!

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
